### ISSUE TITLE
The Six Check Mark Criterias

### ISSUE THREAD

**Issue by DenisCarriere** [EOS-Nation/dac#1]
## The Six Check Mark Criterias

- [ ] 1) Public website URL and at least one social media account (all social media links offered by candidate will be shared in report)
As a public representative of a powerful common resource, a community presence is expected.
- [ ] 2) Links to the following ID information, all posted to the Steem blockchain (either in one post or multiple):
  - [ ] A) Official block producer candidate name.
  - [ ] B) Location of company headquarters.
  - [ ] C) Expected location of servers. 
  - [ ] D) Type of servers (cloud, bare metal, etc).
  - [ ] E) Current employee list and pictures of at least 67% of staff.
  - [ ] F) Relevant background qualifications for at least 67% of staff.

Clear identity of company and employees is important, according to the majority of BP candidates.

- [ ] 3) Estimate of technical specifications and total expenditure for resources by June 3, 2018 - posted to Steem blockchain.
Check mark given for any estimate, modesty encouraged by permanent Steem record. Quality and accuracy of effort judged by the community in public document.
- [ ] 4) Estimated scaling plan for hardware after June 3, 2018 - posted to Steem blockchain.
Rough outlines receive a check mark, open review gauges the effort.
- [ ] 5) Community benefit project outline, only for projects expected to be public by June 3, 2018 - posted to Steem blockchain.
BPs overwhelmingly agreed on the importance of community benefit projects.
- [ ] 6) Listing of Telegram and node names for community testnet participation.
If you still need to join, start at https://t.me/CommunityTestnet
Currently we have 21 candidates signed up on BP candidate form. If you haven't filled out the candidate form you won't be in the report; all candidate names are pulled from the responses.

**Comment by Foutight** [comment]
1) We should share the EOS-Nation facebook group, meaning we should post on there to make our intentions known

**Comment by DenisCarriere** [comment]
Agreed, once we create the Steemit post, we can share on both platforms (and Tweet it)
